src/Controls/tests/Core.UnitTests/AcceleratorTypeConverterUnitTests.cs

```diff
@@ -1,20 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Xunit;
-
-namespace Microsoft.Maui.Controls.Core.UnitTests
-{
-	public class AcceleratorTypeConverterUnitTests : BaseTestFixture
```

Comment:
We can keep this for now, and maybe add some tests to convert from this old type into the new type for compat reasons?